HACKING.rst

```diff
@@ -0,0 +1,111 @@
+=======
+HACKING
+=======
+
+Getting Started
+---------------
+
+Getting the code
+~~~~~~~~~~~~~~~~
+
+If you just want to look at the code, you can just clone the main repository:
+
+::
+    $ git clone git@github.com:juju/theblues.git
```

Comment:
I spoke with @mitechie about it, and we both thought it's fine. Consider it institutional humor.